hal/targets/hal/TARGET_STM/TARGET_STM32L4/device.h

```diff
@@ -0,0 +1,40 @@
+// The 'features' section in 'target.json' is now used to create the device's hardware preprocessor switches.
```

Comment:
yes - 1 file per family instead of 1 per device as they were almost all the same, so this reduces the overall number of files (with the point to maintenance easier) - thanks for having merged  :+1: